### PR TITLE
done homework for lesson-3

### PR DESCRIPTION
**Pull request by ilyabaranowski**

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -37,4 +37,60 @@ for (i = 0; i < arr3.length; i++) {
     if (arr3[i] % 2 == 0 && arr3[i] != 0) {
         console.log(arr3[i])
     }
-}
\ No newline at end of file
+}
+
+//lesson-3--1
+function palindrome(word) {
+    let arr = word.split('').reverse().join('');
```
Comment (ghost) [inline main.js]: You don't plan to change `arr`, use  `const`.

Comment (ghost) [reply main.js]: and result of `word.split('').reverse().join('');` will be `string`, not `arr`. wrong name for variable.

File: main.js [hunk 2]
```diff
@@ -37,4 +37,60 @@ for (i = 0; i < arr3.length; i++) {
...
+    }
+    else {
+        return 'false';
+    }
+}
+console.log(palindrome('Ilya'));
+
+//3--2
+function min(a, b) {
+    if (a < b) {
```
Comment (ghost) [inline main.js:56]: use `?` operator for this, an example on the 66 line.

File: main.js (hunk 1 above)
Comment (ilyabaranowski) [reply main.js]: done

File: main.js (hunk 2 above)
Comment (ilyabaranowski) [reply main.js:56]: i don't understand what should I do. On the 66 line this alredy done.

**Pull request merged**